### ISSUE TITLE
Switch mediawiki from reCaptcha -> hCaptcha

### ISSUE THREAD

**Issue by Firefishy** [openstreetmap/chef#298]
https://www.hcaptcha.com/ support is coming to mediawiki.
This issue is to track that it is an alternative and we should likely switch to it once it becomes fully available in mediawiki / Confirmedit.

https://www.mediawiki.org/wiki/Extension:ConfirmEdit#hCaptcha

**Comment by Tigerfell** [comment]
A removal of CAPTCHA is suggested in openstreetmap/operations#383 and there is a related discussion at https://wiki.openstreetmap.org/wiki/Talk:Wiki#Followup_on_CAPTCHA.

**Comment by miyurusankalpa** [comment]
2 things to note on this.

1. In the linked wiki talk above, user Rostaman is suggests that hCaptcha is owned by cloudflare. which is false. Cloudflare is an hCaptcha user and cloudflare started to use it because Google asked cloudflare to pay a fee for reCaptcha.

https://blog.cloudflare.com/moving-from-recaptcha-to-hcaptcha/

2. hCaptcha is not very user friendly, try it and see. Also read the below community thread in cloudflare asking to stop using hCaptcha.

https://community.cloudflare.com/t/stop-using-hcaptcha/158968

**Comment by grischard** [comment]
If only someone created a 'maptcha' that made people do mini-tasks like the Missing Maps app...

**Comment by tomhughes** [comment]
Yes because encouraging drive by armchair mapping has never caused any problems right ;-)

**Comment by grischard** [comment]
I was thinking something like MapSwipe, where you're one step removed from mappers.

</offtopic>

**Comment by Firefishy** [comment]
I've signed up for a hcaptcha account. We're blocked until we have deployed >= Mediawiki 1.34
